### DISCUSSION TITLE
national weather service API key

### DISCUSSION THREAD

**Discussion by punkmuster** [weather-gov/api#222] [Q&A]
Hello. Very new to home assistant and I'm trying to get the National Weather Service working on home assistant dashboard. Im having trouble getting the API key. When I click on "The API is located at: https://api.weather.gov" on NWS website, it only shows:

{
    "status": "OK"
}

Can someone please help with this? I honestly have no clue but im trying to figure this out and very willing to learn.
Thanks

**Comment by JJKraw** [comment]
Have you read the specification tab/section at https://www.weather.gov/documentation/services-web-api ?

A get at the root (the url you have above) doesn't return any information. That documentation shows what you need to add to the url to get specific info.
